templates/shared_services/admin-vm/template_schema.json

```
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/microsoft/AzureTRE/templates/shared_services/gitea/template_schema.json",
```

The template shouldn't be this path
```suggestion
  "$id": "https://github.com/microsoft/AzureTRE/templates/shared_services/admin-vm/template_schema.json",
```